pkg/antctl/raw/proxy/command_test.go

```diff
@@ -0,0 +1,112 @@
+// Copyright 2023 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package proxy
+
+import (
+	"bytes"
+	"log"
+	"os"
+	"testing"
+
+	"github.com/spf13/afero"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"k8s.io/klog/v2"
+)
+
+func TestValidateAndComplete(t *testing.T) {
+	defaultFS = afero.NewMemMapFs()
+	defer func() {
+		defaultFS = afero.NewOsFs()
+	}()
+
+	_, err := defaultFS.Create("www")
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	tests := []struct {
+		name           string
+		proxyOpts      *proxyOptions
+		expectedErr    string
+		expectedErrLog string
+	}{
+		{
+			name: "specified both port and unixSocket",
+			proxyOpts: &proxyOptions{
+				port:       80,
+				unixSocket: "unix-socket",
+			},
+			expectedErr: "cannot set --unix-socket and --port at the same time",
+		},
+		{
+			name: "specified both controller and agentnodename",
+			proxyOpts: &proxyOptions{
+				port:          80,
+				controller:    true,
+				agentNodeName: "agent-node",
+			},
+			expectedErr: "cannot use --controller and --agent-node at the same time",
+		},
+		{
+			name: "disabled the controller",
+			proxyOpts: &proxyOptions{
+				port:       80,
+				controller: false,
+				staticDir:  "temp",
+			},
+			expectedErrLog: "Failed to stat static file directory\" name=\"temp\" error=\"open temp: file does not exist\"\n",
```

Comment:
nit: I think using "Failed to stat static file directory" as the value of `expectedErrLog` would be good enough here?